exercises/concept/cars-assemble/.meta/design.md

```diff
@@ -6,17 +6,14 @@ The goal of this exercise is to teach the student how to deal with numbers in Go
 
 ## Learning objectives
 
-- This should introduce the `int` and `float64` types (be basic number types in Go).
+- This should introduce the `int` and `float64` types (the basic number types in Go).
```

Comment:
The exercises teaches about `uint` too. It should be added here or alternatively, we should make this exercise not teach `uint`.

Reply:
I've updated `design.md` to add `uint`.